website/content/docs/agent/options.mdx

```diff
@@ -919,7 +919,7 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
     unhealthy. Defaults to 250.
 
   - `min_quorum` - Sets the minimum number of servers necessary
-    in a cluster before autopilot can prune dead servers. There is no default.
+    in a cluster. Autopilot will stop pruning dead servers when this minimum is reached. There is no default.
```

Comment:
This LGTM; 

As I was parsing thru @banks's comment [here](https://github.com/hashicorp/consul/issues/8118#issuecomment-645330040), I was thinking if we could encourage practitioners to set a reasonable default, given N := number of servers